packit/config/package_config_validator.py

```diff
@@ -67,17 +69,31 @@ def validate(self) -> str:
                 f"the file is present."
             )
 
-        if not schema_errors:
-            return f"{self.config_file_path.name} is valid and ready to be used"
+        synced_files_errors = []
+        if config:
+            synced_files_errors = [
+                f for f in iter_srcs(config.synced_files) if not Path(f).is_file()
+            ]
 
         output = f"{self.config_file_path.name} does not pass validation:\n"
-        if isinstance(schema_errors, list):
-            output += "\n".join(map(str, schema_errors))
-            return output
 
-        for field_name, errors in schema_errors.items():
-            output += self.validate_get_field_output(errors, field_name)
-        return output
+        if schema_errors:
+            if isinstance(schema_errors, list):
+                output += "\n".join(map(str, schema_errors))
+            else:
+                for field_name, errors in schema_errors.items():
+                    output += self.validate_get_field_output(errors, field_name)
+
+        if synced_files_errors:
+            output += "The following {} not exist: {}\n".format(
+                ("file do" if (len(synced_files_errors) == 1) else "files does"),
```

Comment:
If the check is changed to `exists()` above, this needs to be updated to "path does"/"paths do".

Reply:
Did a little testing, and here is the output I got when validation fails on missing files:

```
2021-09-21 09:58:17.280 validate_config.py INFO   .packit.yaml does not pass validation:
The following path does not exist: .fmf/
```

I think users might be a little confused by this message, in the sense that they won't immediately know why it's an issue that the path is missing. Please change the message to something like: "The following paths are configured to be synced but do not exist: ".